vllm/compilation/vllm_inductor_pass.py

```diff
@@ -29,6 +30,8 @@ def __init__(self, config: CompilationConfig.PassConfig):
         self.pass_name = self.__class__.__name__
 
     def dump_graph(self, graph: torch.fx.Graph, stage: str, always=False):
+        lazy_format_graph_code(stage, graph.owning_module)
```

Comment:
What's going on here? This is technically private API, so it would be best if we could find another way to do this

Reply:
Nvm, this is pretty stable on the pytorch side.